src/sig/dilithium/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ endif()
 if(OQS_ENABLE_SIG_dilithium_2_aes_avx2 OR OQS_ENABLE_SIG_dilithium_3_aes_avx2 OR OQS_ENABLE_SIG_dilithium_5_aes_avx2)
     add_library(dilithium_common_aes OBJECT pqcrystals-dilithium_common_aes/aes256ctr.c)
     target_include_directories(dilithium_common_aes PRIVATE ${CMAKE_CURRENT_LIST_DIR}/pqcrystals-dilithium_common_aes)
-    target_compile_options(dilithium_common_aes PRIVATE -maes -msse2 -mssse3)
+    target_compile_options(dilithium_common_aes PRIVATE-maes-msse2-mssse3)
```

Comment:
whitespaces, also below

Reply:
I believe this didn't cause issues on my machine is because the only builds that were effected were avx2 optimized implementations. I *believe* it's resolved now, and it looks like the CI is passing.

Reply:
Confirm it's resolved on my machine (which basically is the same as CI). Thanks!